src/components/ConnectorTiles.tsx

```diff
@@ -114,8 +119,16 @@ function ConnectorTiles({
                     query,
                     [columnToSort],
                     searchQuery,
-                    columnToSort,
-                    sortDirection,
+                    [
+                        {
+                            col: CONNECTOR_RECOMMENDED,
+                            direction: 'desc',
+                        },
+                        {
+                            col: columnToSort,
+                            direction: sortDirection,
+                        },
+                    ],
```

Comment:
Now we can pass an array of sorting options. So we always hardcode the sorting of recommended first.